backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,66 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: add-test-correction-columns
+      author: nathan@skylight.digital
+      comment: Adds columns in support of test correction workflow
+      changes:
+        - tagDatabase:
+            tag: add-test-correction-columns
+        - sql:
+            remarks: Create the enumerations needed for tracking test order correction reasons
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.TEST_CORRECTION_TYPE as ENUM('DUPLICATE_TEST', 'INCORRECT_RESULT', 'INCORRECT_TEST_DATE', 'OTHER');
+        - addColumn:
+            tableName: test_order
+            columns:
+              - column:
+                  name: corrected_by
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__test_order__api_user
+                    references: api_user(internal_id)
+              - column:
+                  name: corrected_at
+                  type: DATETIME
+                  constraints:
+                    nullable: true
+              - column:
+                  name: correction_type
+                  type: ${database.defaultSchemaName}.TEST_CORRECTION_TYPE
+                  constraints:
+                    nullable: true
+        - addColumn:
+            tableName: test_event
+            columns:
+              - column:
+                  name: corrected_by
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__test_event__api_user
+                    references: api_user(internal_id)
+              - column:
+                  name: corrected_at
+                  type: DATETIME
+                  constraints:
+                    nullable: true
+              - column:
+                  name: correction_type
+                  type: ${database.defaultSchemaName}.TEST_CORRECTION_TYPE
+                  constraints:
+                    nullable: true
+        - sql:
+            remarks: 
+            sql: |
+              UPDATE ${database.defaultSchemaName}.test_order SET correction_type='OTHER' WHERE reason_for_correction IS NOT NULL;
+              UPDATE ${database.defaultSchemaName}.test_event SET correction_type='OTHER' WHERE reason_for_correction IS NOT NULL;
+              GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
```

Comment:
Looks like this GRANT snuck in from the last database change - should we instead be granting permission on the `correction_type` column?